pkg/apis/sparkoperator.k8s.io/v1beta1/types.go

```diff
@@ -526,6 +526,10 @@ type GPUSpec struct {
 func (s *SparkApplication) PrometheusMonitoringEnabled() bool {
 	return s.Spec.Monitoring != nil && s.Spec.Monitoring.Prometheus != nil
 }
+// VolcanoSchedulingRequired returns true if these executor pods are scheduled by volcano
+func (s *SparkApplication) VolcanoSchedulingRequired() bool {
+	return s.Spec.Executor.SchedulerName != nil && *(s.Spec.Executor.SchedulerName) == "volcano"
```

Comment:
maybe we can change `--enable-volcano-scheduling` to `--batch-scheduler-name`, so we can check executor's schedulerName against batch scheduler name.

Reply:
That's a good point.